modules/Bio/Otter/Lace/Defaults.pm

```diff
@@ -686,7 +686,7 @@ __DATA__
 # disable.
 
 [client]
-url=https://otter.sanger.ac.uk/cgi-bin/otter
+url=http://localhost:8004/cgi-bin/otter
```

Comment:
localhost - is a bad url for production, should be commented with TODO: fix in prod